Updating UKIT 008 (#270)
icub-tech-iit/documentation: docs/upgrade_kits/realsense_holder/support.md

## Upgrade Kit

This upgrade kit is meant to improve iCub's 3D perception and grasping capabilities by adding a RealSense camera plugged on the head, in addition to the other eye cameras.

It ensures the compatibility with more RealSense models.

> |       |       |      |
> | :---: | :---: | :---: |
> | <img src="../img/11.PNG"> | <img src="../img/8.PNG"> | <img src="../img/9.PNG"> |
> |                 |                |         |
> |       |       	          |
> |   :--- |    :-----------           |
> |    IIT alias (used as ordering reference)| KIT_008  |

Reviewer: ```suggestion
|  |  |  |
| :---: | :---: | :---: |
| <img src="../img/11.PNG"> | <img src="../img/8.PNG"> | <img src="../img/9.PNG"> |

|  |  |
| :--- | :--- |
|  IIT alias (used as ordering reference) | KIT_008  |
```

## Upgrade Kit

This upgrade kit is meant to improve iCub's 3D perception and grasping capabilities by adding a RealSense camera plugged on the head, in addition to the other eye cameras.

It ensures the compatibility with more RealSense models.

|       |       |      |
| :---: | :---: | :---: |

> | <img src="img\11.PNG" height=250 > | <img src="img\8.PNG" height=250 > | <img src="img\9.PNG" height=250 > |

Reviewer: We need forward slashes here.

Author: Corrected @pattacini

|   :---: |    :-----------: |:---:|     :------------: |   :---:   |
|    1   |	INTEL_REALSENSE_D435i  |  |Intel Realsense D435i with IMU, range from 0.2 to 10m, FOV 90 de | 14567 |

> **Note 1**
> 

> > [Logic schematics](https://github.com/icub-tech-iit/electronics-wiring-public/tree/master/icub-upgrade-kits/kit_008/15389%20-%20iCub2.5%20Head%20RealSense%20upgrade%20kit%20(electronic%20parts)) 

Reviewer: Wrong

|	INTEL_REALSENSE_D405 | 	Intel RealSense D405 with IMU, range from 7 cm to 50 cm, FOV 87°×58°  | 17231 |

**NOTE**: Not all the RealSense cameras have the same EKIT material, some of them connect by using a different cable.

|     RealSense model        |  associated cable      | Cable Wgst Code |
|    :-----------: |     :------------: |   :---:   |
|	INTEL_REALSENSE_D415   |USB Shielded I/O Cable Assembly, Type C/M to C/M, USB 3.1 GEN4, 3.0m Length, Black | 	15365 |
|	INTEL_REALSENSE_D435i   |USB Shielded I/O Cable Assembly, Type C/M to C/M, USB 3.1 GEN4, 3.0m Length, Black | 	15365 |
|	INTEL_REALSENSE_D405   |USB Shielded I/O Cable Assembly, USB 3.2 Gen 1, Male A to Male Micro-B, 3.0m length, black | 17308 |

## Assembly instruction

The RealSense holder can be mounted on the top of iCub head by replacing two side screws fixing the face cover, and in the kit are included and USB Type-C cable, screws and cable ties. The kit does NOT include the RealSense. It is recommended to replace the short cable included in the RealSense retail package with the longer one provided with the upgrade kit.

> **NOTE**: The following instructions are provided for assembling the D435i model of RealSense camera.

Author: The title is out of the admonition box. What I am doing wrong @pattacini? I'm following this format:  
  
!!! note "Phasellus posuere in sem ut cursus" Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla et euismod nulla. Curabitur feugiat, tortor non consequat finibus, justo purus auctor massa, nec semper lorem quam in massa.

> **NOTE**: The following instructions are provided for assembling the D435i model of RealSense camera.

Reviewer: Before it wasn't correct. I've seen you fixed it in the latest commits.

 <img src="img/6.PNG" height=400 alignement=center >

**Assembly sequence**

*Perform the assembly sequence as shown in the following images.*

- First step, remove the two screws V4-12--_-_U5933_C from the side of the face cover as can be seen in the image. Set them aside, just in case you want to successively unmount the RealSense holder.

>  <img src="img/2.PNG" height=400 alignement=center >

Reviewer: Did you check that the alignment specified this way actually works?

Author: I tried something other but didn't work. For now, I'm not centring the pictures anymore.

Reviewer: We need to center images. So, please stick to the previous format that was relying on `<center>...</center>`.

Author: Re-centered pictures @pattacini

- Second step, fix the holder IG_027_P_002 with the respective screws V4-16--_-_ISO7045_CZ as shown in the image, and fix the hook to the backward cover air vents.

 <img src="img/3.PNG" height=400 alignement=center >

- Third step, fix the chosen RealSense model to the holder usign the screws V3-8--_-_U5933_C. At this point is possible to plug the USB Type-C cable cod. 15365 and use the cable ties cod. 531 to fix it around the head.

 <img src="img/4.PNG" height=400 alignement=center >
 <img src="img/5.PNG" height=400 alignement=center >

>  **NOTE:** As mentioned above, the RealSense D405 uses a different cable too.

Reviewer: Same comment as above.

|   1     |    15365 	 |      	 | USB Shielded I/O Cable Assembly, Type C/M to C/M, USB 3.1 GEN4, 3.0m Length, Black | 15365 |
|   6     |    531       |           |  Cable tie, 100x2.5mm, nylon, black | 531 | 

 !!! note  "Note 2"

     See also [historical review](https://github.com/Mick3Lozzo/documentation/blob/master/docs/upgrade_kits/realsense_holder/support.md#historical-review) for further details. 

!!! note  "Note 3"

    See [Harness schematics](https://github.com/icub-tech-iit/electronics-wiring-public/tree/master/icub-upgrade-kits/kit_008/15393%20-%20iCub2.5%20Head%20RealSense%20upgrade%20kit%20(wiring%20parts)) 

## Historical review
This upgrade kit has been developed together with the [HSP@IIT](https://www.iit.it/web/humanoid-sensing-and-perception) research line as an optional plug-in support to improve iCub perception and facilitate it in grasping tasks iCub. For reference, see also the page [design holder realsense](https://github.com/icub-tech-iit/design-holder-realsense).

<center> <img src="img/1.PNG" height=400> </center>

> n designed
> The holder has bee to be compatible with the following RealSense models:

Reviewer: ```suggestion
The holder has been designed to be compatible with the following RealSense models:
```

!!! note  "Note 5"

    The following instructions are provided for assembling the D435i model of RealSense camera.

<center> <img src="img/6.PNG" height=400> </center>

**Assembly sequence**

*Perform the assembly sequence as shown in the following images.*

- First step, remove the two screws V4-12--_-_U5933_C from the side of the face cover as can be seen in the image. Set them aside, just in case you want to successively unmount the RealSense holder.

<center> <img src="img/2.PNG" height=400> </center>

- Second step, fix the holder IG_027_P_002 with the respective screws V4-16--_-_ISO7045_CZ as shown in the image, and fix the hook to the backward cover air vents.

<center> <img src="img/3.PNG" height=400> </center>

- Third step, fix the chosen RealSense model to the holder usign the screws V3-8--_-_U5933_C. At this point is possible to plug the USB Type-C cable cod. 15365 and use the cable ties cod. 531 to fix it around the head.

<center> <img src="img/4.PNG" height=400> </center>

> <img src="img/5.PNG" height=500>

Reviewer: ```suggestion
<center> <img src="img/5.PNG" height=500> </center>
```

Reviewer: You could commit the suggestion straight away @Mick3Lozzo.

|   1     |    15365 	 |      	 | USB Shielded I/O Cable Assembly, Type C/M to C/M, USB 3.1 GEN4, 3.0m Length, Black | 15365 |
|   6     |    531       |           |  Cable tie, 100x2.5mm, nylon, black | 531 | 

> **Note 2**
> 
> See also [historical review](https://github.com/Mick3Lozzo/documentation/blob/master/docs/upgrade_kits/realsense_holder/support.md#historical-review) for further details. 

> **Note 3**
> 

> > See [Harness schematics](https://github.com/icub-tech-iit/electronics-wiring-public/tree/master/icub-upgrade-kits/kit_008/15393%20-%20iCub2.5%20Head%20RealSense%20upgrade%20kit%20(wiring%20parts)) 

Reviewer: Wrong

The RealSense holder can be mounted on the top of iCub head by replacing two side screws fixing the face cover, and in the kit are included and USB Type-C cable, screws and cable ties. The kit does NOT include the RealSense. It is recommended to replace the short cable included in the RealSense retail package with the longer one provided with the upgrade kit.

> **Note 4**
>
>   The following instructions are provided for assembling the D435i model of RealSense camera.
> 

> > <img src="img/6.PNG" height=400 alignement=center >

Reviewer: Wrong
